numbers_letters_and_variables/number_of_seconds.rb

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+SEC_PER_MIN = 60
```

Comment:
In Ruby, constants are specified by all upper case letters.  Also, please be more explicit in your variable names, i.e. `seconds_per_minute`.